src/scheduled_tasks.rs

```diff
@@ -320,6 +319,16 @@ async fn clear_old_activities(pool: &mut DbPool<'_>) {
   }
 }
 
+async fn delete_old_denied_users(pool: &mut DbPool<'_>) {
+  LocalUser::delete_old_denied_local_users(pool)
+    .await
+    .map(|_| {
+      info!("Done.");
+    })
+    .map_err(|e| error!("Failed to deleted old denied users: {e}"))
```

Comment:
`.inspect()` was stabilized in the latest Rust version so you could use that.

Reply:
Neat, updated the rest of these too now.

Reply:
Actually it doesnt really make any difference.